MAVProxy/mavproxy.py

```diff
@@ -280,6 +304,8 @@ def unload_module(modname):
             if hasattr(m, 'unload'):
                 m.unload()
             mpstate.modules.remove((m,pm))
+            mpstate.modules_idle.remove((m,pm))
```

Comment:
would a silent failure be best in here, or should we explicitly check for existance before calling remove?